sklearn/decomposition/pca.py

```diff
@@ -120,25 +124,68 @@ class PCA(BaseEstimator, TransformerMixin):
 
             n_components == min(n_samples, n_features)
 
-        if n_components == 'mle', Minka\'s MLE is used to guess the dimension
-        if ``0 < n_components < 1``, select the number of components such that
-        the amount of variance that needs to be explained is greater than the
-        percentage specified by n_components
+        if n_components == 'mle' and svd_solver == 'full', Minka\'s MLE is used
+        to guess the dimension
+        if ``0 < n_components < 1`` and svd_solver == 'full', select the number
+        of components such that the amount of variance that needs to be
+        explained is greater than the percentage specified by n_components
+        n_components cannot be equal to n_features for svd_solver == 'arpack'.
 
-    copy : bool
+    copy : bool (default False)
```

Comment:
It's `True` by default: we don't want to silently / implicitly center the input data provided by the user. The code is right. Only the docstring need a fix.